<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleProductItem</class>
 <widget class="QWidget" name="SimpleProductItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>200</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 237, 237)</string>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="rootFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="4" column="3">
       <widget class="QLabel" name="newPriceLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2" rowspan="2">
       <widget class="QLabel" name="discountLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white; 
      background-color: rgb(110, 29, 114); 
      border-radius: 3px;
      padding: 10px;</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2" alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QPushButton" name="wishlistButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/heart.svg</normaloff>:/icons/heart.svg</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="coverLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>116</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>116</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="3" column="4" rowspan="2">
       <widget class="QPushButton" name="addToCartButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(180, 230, 60), stop:1 rgb(140, 202, 39));</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/cart-plus.svg</normaloff>:/icons/cart-plus.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="oldPriceLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <strikeout>true</strikeout>
         </font>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
